contrib/screener-api/screener/screener.go

```diff
@@ -65,43 +60,61 @@ func NewScreener(ctx context.Context, cfg config.Config, metricHandler metrics.H
 	docs.SwaggerInfo.Title = "Screener API"
 	docs.SwaggerInfo.Host = fmt.Sprintf("localhost:%d", cfg.Port)
 
-	screener.client, err = trmlabs.NewClient(cfg.TRMKey, core.GetEnv("TRM_URL", "https://api.trmlabs.com"))
-	if err != nil {
-		return nil, fmt.Errorf("could not create trm client: %w", err)
-	}
-	screener.thresholds = cfg.VolumeThresholds
+	screener.client = chainalysis.NewClient(
+		cfg.RiskLevels, cfg.ChainalysisKey, core.GetEnv("CHAINALYSIS_URL", cfg.ChainalysisURL))
 
+	screener.blacklistCache = make(map[string]bool)
 	for _, item := range cfg.Whitelist {
-		screener.whitelist = append(screener.whitelist, strings.ToLower(item))
-	}
-
-	screener.rulesManager, err = setupScreener(cfg.Rulesets)
-	if err != nil {
-		return nil, fmt.Errorf("could not setup screener: %w", err)
+		screener.whitelist[strings.ToLower(item)] = true
 	}
 
 	dbType, err := dbcommon.DBTypeFromString(cfg.Database.Type)
 	if err != nil {
 		return nil, fmt.Errorf("could not get db type: %w", err)
 	}
-
 	screener.db, err = sql.Connect(ctx, dbType, cfg.Database.DSN, metricHandler)
 	if err != nil {
 		return nil, fmt.Errorf("could not connect to rules db: %w", err)
 	}
 
 	screener.router = ginhelper.New(logger)
 	screener.router.Use(screener.metrics.Gin())
-	screener.router.Handle(http.MethodGet, "/:ruleset/address/:address", screener.screenAddress)
```

Comment:
we need to have a /:ruleset/address/:address here where ruleset is a no-op for backwards compatibility

Reply:
in practice this is just:

```go
   // TODO: deprecated and ruelset is a no-op, this is for backwards compatibility and may be removed in a future version.
    screener.router.Handle(http.MethodGet, "/:ruleset/address/:address", screener.screenAddress)
```